salt/modules/zypper.py

```diff
@@ -649,10 +649,10 @@ def version_cmp(ver1, ver2, ignore_epoch=False):
     return __salt__['lowpkg.version_cmp'](ver1, ver2, ignore_epoch=ignore_epoch)
 
 
-def list_pkgs(versions_as_list=False, **kwargs):
+def list_pkgs(versions_as_list=False, attr=None, **kwargs):
```

Comment:
@damon-atkins since the topic is not about "standard signature" anymore, I see very little difference here:

```python
>>> def foo(a=None, **kw): print a, kw
>>> foo(**{'a':'Foo', 'b':'bar'})
Foo {'b': 'bar'}
>>> foo(a='Foo', b='bar')
Foo {'b': 'bar'}
>>> foo(b='bar')
None {'b': 'bar'}
>>>
>>> def foo(**kw): print 'a' in kw and kw.pop('a') or None, kw
>>> foo(a='Foo', b='bar')
Foo {'b': 'bar'}
>>> foo(**{'a':'Foo', 'b':'bar'})
Foo {'b': 'bar'}
>>> foo(b='bar')
None {'b': 'bar'}
```

The only why one would have `attr` inside the `kwargs` is that the code itself takes care of it, or just passed elsewhere, like in case of `info_installed`. I see a very little merit discussing all that.